Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Dragons overhaul

## Changelog:
**Aspects traits**

- [ ] - _Aspect of Life_: Prowess bonuses, diplomacy, steward and learning bonuses, life and fire magic bonuses (when magic lifestyle is done)

- [ ] - _Aspect of Nature_: Prowess bonuses, diplomacy, steward and learning bonuses, huge life magic bonuses (when magic lifestyle is done)

- [ ] - _Aspect of Earth_: Huge prowess bonuses (Aspect of Earth was made to protect through combat.), martial and intrigues bonuses, earth and fire magic bonuses (when magic lifestyle is done)

- [ ] - _Aspect of Magic_: Prowess bonuses, learning and steward bonuses, huge arcane magic bonuses (when magic lifestyle is done), more health

- [ ] - _Aspect of Time_: Prowess bonuses, learning and steward bonuses, arcane (and another for balancing ?) magic bonuses (when magic lifestyle is done), more health

**Others Aspects traits**

- [ ] - _Aspect of Death (Corrupted Aspect of Earth)_: Huge prowess bonuses (Inherited from Aspect of Earth), martial and intrigues bonuses, fire magic bonuses, huge void magic bonuses (when magic lifestyle is done), more health

- [ ] - _Aspect of Infinite (Corrupted Aspect of Infinite)_: Prowess bonuses , learning and steward bonuses, arcane magic bonuses, huge void magic bonuses (when magic lifestyle is done), more health

**Dragonflight government**

- [ ] - _Troops bonuses and dynastic opinion maluses_: As in CK2's Hive government, in this type of government you should have a bonus of troops per child and a dynastic opinion penalty.

- [ ] - _Dragonflight succession_: The succession of a dragonflight should be done by electing the best dragon in the flight, with choice bonuses for opinion. Some should have title claims. Which would explain why Wrathion would become the self-proclaimed leader of the black flight after the fall of Deathwing.

- [ ] - _Dragonflight succession event_: When the succession is done, there should be an event / decision that brings all the flight together for the ritual of creating a new aspect. As a result, the new leader of the dragonflight would gain the title of aspect.

- [ ] - _Brood special contract_: We should be able to create broods, which are less defended but which bring big bonuses of troops and fertility for the overlord of the brood.

**Draconic faiths**

- [ ] - _Green faith_:  Faith about nature things and protect that. Should have life syncretism. If we put back the emerald dream system, they should have wide access to it. And be subject to the corruption of the void.

- [ ] - _Black faith_: Faith about protect azeroth. Replaced in Blackdragonflight by N'Zoth worship. Except maybe for Ebonhorn ? Can be come back if we put an uncorruption system.

- [ ] - _Red faith_: Faith about life things and protect that. Should have life syncretism.

- [ ] - _Blue faith_: Faith about magic things and protect that. They should be wary of mortals who practice magic.

- [ ] - _Bronze faith_: Faith about time things and protect that. If we put back the emerald dream system, they should have a system like that for going through the corridors of time. And be subject to the corruption of the void.

**Special things**

- [ ] - _Dragonspawn/Drakonide transformation_: Mortals who practice the religions above should be able, at some point, to transform into dragonspawn or drakonid. Like on a pilgrimage to the shrines in Dragonblight. Once this is done, they become loyal to the dragonflight, perhaps automatic vassalization? If they are of imperial title, they simply ally themselves with theft.

- [ ] - _Black dragons corruption_ : Black dragons must worship the Old Gods no matter what. We can do a decorruption system later. For Ebonhorn and Wrathion for example. This is done "biologically" and not through education. Since it is only the eggs that have been decayed that are not under the influence of the Old Gods.

# How to test:
- Open the game
- Check the changes
